app/assets/stylesheets/templatemo-style.scss

```diff
@@ -0,0 +1,403 @@
+    0% {
+        -webkit-transform: perspective(120px) rotateX(0deg) rotateY(0deg);
+        transform: perspective(120px) rotateX(0deg) rotateY(0deg);
+    }
+    50% {
+        -webkit-transform: perspective(120px) rotateX(-180.1deg) rotateY(0deg);
+        transform: perspective(120px) rotateX(-180.1deg) rotateY(0deg);
+    }
+    100% {
+        -webkit-transform: perspective(120px) rotateX(-180deg) rotateY(-179.9deg);
+        transform: perspective(120px) rotateX(-180deg) rotateY(-179.9deg);
+    }
+}
+@keyframes sk-rotatePlane {
+    0% {
+        -webkit-transform: perspective(120px) rotateX(0deg) rotateY(0deg);
+        transform: perspective(120px) rotateX(0deg) rotateY(0deg);
+    }
+    50% {
+        -webkit-transform: perspective(120px) rotateX(-180.1deg) rotateY(0deg);
+        transform: perspective(120px) rotateX(-180.1deg) rotateY(0deg);
+    }
+    100% {
+        -webkit-transform: perspective(120px) rotateX(-180deg) rotateY(-179.9deg);
+        transform: perspective(120px) rotateX(-180deg) rotateY(-179.9deg);
+    }
+}
+/* end preloader */
+/* start navigation */
+
+.navbar-default {
+    background: #fff;
+    border: none;
+    box-shadow: 0px 2px 8px 0px rgba(50, 50, 50, 0.08);
+    margin: 0!important;
+}
+.navbar-default .navbar-brand {
+    color: #28a7e9;
+    font-size: 30px;
+    font-weight: bold;
+    height: 70px;
+    line-height: 35px;
+}
+.navbar-default .nav li a {
+    color: #666;
+    font-size: 13px;
+    font-weight: bold;
+    height: 70px;
+    line-height: 40px;
+}
+.navbar-default .nav li a:hover,
+.navbar-default .nav li a:focus,
+.navbar-default .nav li a.current {
+    color: #28a7e9;
+}
+.navbar-default .navbar-toggle {
+    border: none;
+    padding-top: 20px;
+}
+.navbar-default .navbar-toggle .icon-bar {
+    background: #28a7e9;
+    border-color: transparent;
+}
+.navbar-default .navbar-toggle:hover,
+.navbar-default .navbar-toggle:focus {
+    background-color: transparent
+}
+/* end navigation */
+/* start home */
+
+#home {
+    background: image_url('home-bg.jpg') no-repeat;
+    background-size: cover;
+    color: #fff;
+    margin-top: 70px;
+    text-align: center;
+    width: 100%;
+}
+#home p {
+    font-weight: 400;
+    font-style: italic;
+    line-height: 2em;
+    font-size: 1.2em;
+}
+#home img {
+    display: inline-block;
+    margin-top: 30px;
+}
+/* end home */
+/* start divider */
+
+#divider {
+    text-align: center;
+    padding-top: 80px;
+    padding-bottom: 80px;
+}
+#divider .fa {
+    color: #28a7e9;
+    font-size: 60px;
+}
+#divider h3 {
+    font-size: 20px;
+}
+/* end divider */
+/* start feature */
+
+.feature-even {
+  background: #f8f8f8;
+}
+
+/* end feature */
+/* start feature1 */
+
+.feature-block {
+  padding-top: 80px;
+  padding-bottom: 80px;
+
+  p {
+    padding-top: 10px;
+  }
+
+  span {
+    float: left;
+  }
+
+  .fa {
+    background: #28a7e9;
+    border-radius: 50%;
+    color: #fff;
+    display: inline-block;
+    width: 40px;

```

Comment:
Properties should be ordered background, border-radius, color, display, height, line-height, margin-right, text-align, width